docs/guides/intersphinx.rst

```diff
@@ -0,0 +1,143 @@
+Link to Other Project's Documentation With Intersphinx
+======================================================
+
+You may be familiar with using the :ref:`:ref: role <sphinx:ref-role>` to link to any location of your docs.
+It helps you to keep all links up to date, Sphinx will raise a warning if the reference doesn't exist.
+
+Some times you may need to link to a location of another documentation project.
+We could just link to where the documentation is hosted,
+and use Sphinx's ``linkcheck`` builder to check for broken links.
+
+Another way is using :doc:`Intersphinx <sphinx:usage/extensions/intersphinx>`.
+Which allows you to use all :ref:`cross-reference roles <sphinx:xref-syntax>` from Sphinx with objects in other projects.
+This is, you could use the ``:ref:`` role to link to sections of other documentation projects.
+
+Using Intersphinx
+-----------------
+
+To use Intersphinx you need to add it to the list of extensions in your ``conf.py`` file.
+
+
+.. code:: python
+
+   # conf.py file
+
+   extensions = [
+       'sphinx.ext.intersphinx',
+   ]
+
+And use the ``intersphinx_mapping`` configuration to indicate the name and link of the projects you want to use.
+
+.. code:: python
+
+   # conf.py file
+
+   intersphinx_mapping = {
+       'sphinx': ('https://www.sphinx-doc.org/en/master/', None),
+   }
+
+
+Now we can use the ``sphinx`` name with a cross-reference role:
+
+.. code:: rst
+
+   - :ref:`sphinx:ref-role`
+   - :ref:`:ref: role <sphinx:ref-role>`
+   - :doc:`sphinx:usage/extensions/intersphinx`
+   - :doc:`Intersphinx <sphinx:usage/extensions/intersphinx>`
+
+Result:
+
+- :ref:`sphinx:ref-role`
+- :ref:`:ref: role <sphinx:ref-role>`
+- :doc:`sphinx:usage/extensions/intersphinx`
+- :doc:`Intersphinx <sphinx:usage/extensions/intersphinx>`
+
+Intersphinx in Read the Docs
+----------------------------
+
+You can use Intersphinx to link to subprojects, translations, another version or any other project hosted in Read the Docs.
+For example:
+
+.. code:: python
+
+   # conf.py file
+
+   intersphinx_mapping = {
+       # Links to "v2" version of the "docs" project.
+       'docs-v2': ('https://docs.readthedocs.io/en/v2', None),
+       # Links to the French translation of the "docs" project.
+       'docs-fr': ('https://docs.readthedocs.io/fr/latest', None),
+       # Links to the "apis" subproject of the "docs" project.
+       'sub-apis': ('https://docs.readthedocs.io/projects/apis/en/latest', None),
+   }
+
+Intersphinx with private projects
+---------------------------------
+
+If you are using :doc:`/commercial/index`,
+Intersphinx will not be able to fetch the inventory file from private docs.
+
+Intersphinx supports `URLs with Basic Authorization <https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#using-intersphinx-with-inventory-file-under-basic-authorization>`__,
```

Comment:
sphinx doesn't have a link to this section (actually I added that section :grin:) and they don't use the `autosectionlabel` extension :(